core/src/autogluon/core/utils/infer_utils.py

```diff
@@ -22,6 +23,8 @@ def get_model_true_infer_speed_per_row_batch(
         If simulating large-scale batch inference, values of 100000+ are recommended to get genuine throughput estimates.
     repeats : int, default = 1
         Repeats of calling leaderboard. Repeat times are averaged to get more stable inference speed estimates.
+    persist_models : bool, default = True
```

Comment:
Didn't see this argument passed in the argument. Is this intentional?

If this is intentional, `persist_models` might be undefined when we are trying to get its value.

Reply:
yeah thanks, I forgot to commit this one line

Reply:
added